UnitTests/ConsoleDriverTests.cs

```diff
@@ -66,5 +68,164 @@ public void SetColors_Changes_Colors ()
 			Assert.Equal (ConsoleColor.Black, Console.BackgroundColor);
 			driver.End ();
 		}
+
+		[Fact]
+		public void FakeDriver_Always_Sends_ConsoleKey_Oem3_Without_Using_MockKeyPresses ()
+		{
+			Application.Init (new FakeDriver (), new FakeMainLoop (() => FakeConsole.ReadKey (true)));
+
+			var top = Application.Top;
+			var view = new View ();
+			var oem3 = 0;
+
+			view.KeyPress += (e) => {
+				Assert.Equal ('~', (uint)e.KeyEvent.Key);
```

Comment:
Did you miss this?

Reply:
Yes, due the constant errors, I forgot to check this, sorry. I searched for '~' in the `UnitTests` and found none.